src/useInteractionManager.ts

```diff
@@ -5,9 +5,14 @@ export function useInteractionManager() {
   const [complete, updateInteractionStatus] = useState(false)
 
   useEffect(() => {
-    InteractionManager.runAfterInteractions(() => {
+    const {cancel} = InteractionManager.runAfterInteractions(() => {
       updateInteractionStatus(true)
     })
+
+    return () => {
+      cancel?.()
```

Comment:
`runAfterInteractions` returns a promise and I'm not sure if they are required to have their methods bound so that they work without `this` being set 🤔 

Also, I think that `cancel` will always be set, right? It's documented as returning a cancelable promise 🤔 

What do you think about this instead:

```suggestion
    const promise = InteractionManager.runAfterInteractions(() => {
      updateInteractionStatus(true)
    })

    return () => {
      promise.cancel()
```